modules/pam_namespace/pam_namespace.c

```diff
@@ -870,6 +871,12 @@ static char *md5hash(const char *instname, struct instance_data *idata)
      * Create MD5 hashes for instance pathname.
      */
 
+    len = strlen(instname);
+    if (len > UINT_MAX) {
+        pam_syslog(idata->pamh, LOG_CRIT, "Instance pathname too long");
+        return NULL;
+    }
```

Comment:
Likewise.

Reply:
In fact, this check is redundant. Even though md5hash itself does not check, its only caller makes sure that instname is not that long. Dropped.